site/docs/tools/deltalake/spark.md

```diff
@@ -0,0 +1,195 @@
+# Spark via Delta Lake
+
+!!! note
+    Detailed steps on how to set up Pyspark + Delta Lake + Nessie with Python is available on [Binder](https://mybinder.org/v2/gh/projectnessie/nessie-demos/main?filepath=notebooks/nessie-delta-demo-nba.ipynb).
+
+To access Nessie from a spark cluster make sure the `spark.jars` spark option is set to include
+the [Spark 2](https://repo.maven.apache.org/maven2/org/projectnessie/nessie-deltalake-spark2/{{ versions.java}}/nessie-deltalake-spark2-{{ versions.java}}.jar)
+or [Spark 3](https://repo.maven.apache.org/maven2/org/projectnessie/nessie-deltalake-spark3/{{ versions.java}}/nessie-deltalake-spark3-{{ versions.java}}.jar)
+jar. These jars contain all Nessie **and** Delta Lake libraries required for operation.
+
+!!! note
+    the `spark3` jar is for Delta versions >0.7.0 on Spark3 and the `spark2` jar is for Delta versions 0.6.x on Spark2.4
+
+In pyspark this would look like
+
+``` python
+SparkSession.builder
+    .config('spark.jars.packages',
+            'org.projectnessie:nessie-deltalake-spark3:{{ versions.java}}')
```

Comment:
Guess, that's all maybe stuff for a separate PR (to clean up this DL/Spark page).